logicrunner/rpc_methods_test.go

```diff
@@ -402,11 +403,11 @@ func TestSaveAsChildRegistersOutgoingRequestWithValidReason(t *testing.T) {
 	outgoingReqID := gen.ID()
 	outgoingReqRef := insolar.NewReference(outgoingReqID)
 	// Make sure an outgoing request is registered
-	am.RegisterOutgoingRequestMock.Set(func(ctx context.Context, r *record.OutgoingRequest) (*insolar.ID, error) {
+	am.RegisterOutgoingRequestMock.Set(func(ctx context.Context, r *record.OutgoingRequest) (*payload.RequestInfo, error) {
 		require.Nil(t, outreq)
 		outreq = r
 		id := outgoingReqID
-		return &id, nil
+		return &payload.RequestInfo{RequestID: id}, nil
```

Comment:
As @ruz mentioned we need some new tests for the feature.

Reply:
yep, agree